content/docs/user-guide/scripting/script-canvas/programmer-guide/custom-nodes/_index.md

```diff
@@ -5,292 +5,39 @@ description: Learn how to create custom Script Canvas nodes in Open 3D Engine (O
 weight: 200
 ---
 
-Creating custom nodes in Script Canvas offers you maximum control and flexibility with the functionality of the node. You might wish to a create custom node in the following scenarios:
+## Feature overview
+Creating custom nodes in Script Canvas offers you maximum control and flexibility with the functionality of the node. 
+You might wish to a create custom node in the following scenarios:
 
 * When your node has state, time, or latent results.
 * When creating complex nodes.
 * When you need control over the node's topology.
+* When you want to reflect your own C++ free functions. 
 
-To make the process of creating a custom node easier, Script Canvas uses a templated, automatic code generation system called [AzAutoGen](/docs/user-guide/programming/autogen/) to significantly reduce the amount of "boilerplate code" you need to write just to get a node up and running, allowing the developer to immediately focus on the functionality of the new node, since the code required for the node to show up in the **Node Palette** is already there.
+To make the process of creating a custom node easier, Script Canvas uses a templated, automatic code generation system called [AzAutoGen](/docs/user-guide/programming/autogen/)
+to significantly reduce the amount of "boilerplate code" you need to write just to get a node up and running, allowing the developer to immediately focus on the functionality of the new node,
+since the code required for the node to show up in the [Node Palette](/docs/user-guide/scripting/script-canvas/get-started/concepts-and-terms/#node-palette) is already there.
 
 The following four steps are required to create a custom node:
 
 1. Create a code generation XML file.
-1. Create the C++ class files for your node.
-1. Add these files to CMake.
-1. Reflect the new node by adding it to a library.
+2. Create the C++ files for your node.
```

Comment:
FYI, it's a preferred style to use auto-numbering here. It makes it easier to insert steps later in some editors. You might need to tell your particular editor to stop changing the auto-numbering to fixed numbering.

Reply:
done